include/ck/utility/block_to_ctile_map.hpp

```diff
@@ -0,0 +1,258 @@
+#ifndef UTILITY_BLOCK_TO_CTILE_MAP
+#define UTILITY_BLOCK_TO_CTILE_MAP
+
+#include "utility/math.hpp"
+#include "utility/number.hpp"
+#include "tensor_description/tensor_adaptor.hpp"
```

Comment:
Because this header depends on file in ```tensor_description/```, it should not be inside ```utility```, please but it into ```tensor_operation/gpu/grid/```

Reply:
file moved to tensor_operation/gpu/grid